### ISSUE TITLE
Use Cases: Proposal for Organizing Use Cases Submission

### ISSUE THREAD

**Issue by santosomar** [oasis-tcs/openeox#4]
Originally, tracked at:

https://github.com/OpenEoX/openeox/issues/26

**Comment by santosomar** [comment]
I agree with the proposal from @sparrell - originally at https://github.com/OpenEoX/openeox/issues/26

**Creation of Use Cases Repository/Subdirectory:**
   Establish a dedicated repository or a subdirectory within an existing repo specifically for the submission of use cases by contributors.

**Non-Refutation Policy:**
Adopt a process where submitted use cases are not refuted or dismissed to foster an inclusive and innovative environment.

**Personal Experience Submissions:**
Contributors should submit use cases based on their own experiences and not on behalf of others or hypothetical situations.

**Structure for Submissions:**
   - **Contributor Use Cases Subdirectory:** Each contributor can submit their use cases to their own subdirectory (e.g., `usecases/contributor/sFractal`).
   - **General Use Cases Subdirectory:** A separate directory for use cases that are agreed upon as general enough, potentially after editing.

**Action Items:**

- Discuss and finalize the structure of the repository/subdirectories during a TC meeting.
- Establish guidelines for submission based on the proposal.

**Objective:**

The proposal aims to create a collaborative and efficient environment for collecting a diverse set of use cases, helping us to better understand and address the varied contexts in which our upcoming standard operates.

**Comment by sparrell** [comment]
I was the original proposer and would like ot modify my submission. I propose to change "use case" to "value scenario". 

Personally I think term "use case" is ok in that I mean a case where the thingie is used. However in other meetings where I use this term, the PhD computer scientists tend to think I mean something more formal and that use case has a formal definition in computer science. I spent the a decade back in the last century writing formal computer science use cases for the USG using Rational Rose and they are great for requirements work but not so good the average person understanding the value to them.

It came up again in a meeting Monday on a very closely related topic (CISA work stream on VEXs) so I would like to avoid the confusion. When I say "value scenario" I mean something like https://github.com/opencybersecurityalliance/casp/blob/main/Plugfests/2023-06-13-USC/UseCases/README.md#the-whitchywashy-zero-day and not something like https://www.cs.sjsu.edu/faculty/pearce/modules/lectures/ooa/requirements/UseCaseDiagrams.htm.

At the highest level, the difference is subtle but important. The difference is the value scenario shows the outside world the value of operation vs the computer science use case shows the requirements on how the system will work. I think the "value scenario" is more suited for 'awareness & adoption' and reading by execs and business types.

If someone has a better term that 'value scenario', please propose it.

**Comment by sparrell** [comment]
Wrt "Contributors should submit use cases based on their own experiences and not on behalf of others or hypothetical situations.":

- I strongly agree with the "not on behalf of others". Contributors should submit from their persepective and not speak for others. Other groups have went down some edge-case rabbit-holes that may have been submitted just to dork the process. They started with "what if someone wants ....". All our scenarios should be authored by those who actually need whatever ie "our organization ...."
- I disagree with "or hypothetical situations". I think I understand what was meant, but I think it is too strongly worded. For example, sFractal Consulting (ie I'm speaking first person per previous bullet) would like EoX scenarios to include how end-of-life/support is designated if sFractal goes out of business. It is a hypothetical example (at least I hope so :-) ) but a valid scenario - at least for the 'contributor' subdirectory. The group may decide there is not consensus to move company-out-of-business into the general subdir, but I can still put it in the sFractal subdir of contributor value scenarios.

**Comment by sparrell** [comment]
I created pull request #5 to show what I was thinking of (albeit I haven't added any sFractal actual scenarios yet - just the structure)

**Comment by tschmidtb51** [comment]
TC meeting on 2025-10-15: This is resolved through the use cases added earlier this year.
